pkg/scaffold/v1/manager/cmd.go

```diff
@@ -46,14 +46,15 @@ import (
 	"flag"
 	"os"
 
+	"{{ .Repo }}/pkg/apis"
+	"{{ .Repo }}/pkg/controller"
+	"{{ .Repo }}/pkg/webhook"
+
```

Comment:
I would leave this as a separate group but at the end. Reason: these imports are not those of kubebuilder, but of the package we are scaffolding, and thus `{{ .Repo }} ...` imports are the local package ones while the rest are external imports.